moped-editor/src/utils/stringToTitleCase.js

```diff
@@ -0,0 +1,7 @@
+export default str => {
+  let lowerStr = str.toLowerCase();
+  // uppercase first character
+  lowerStr = lowerStr.charAt(0).toUpperCase() + lowerStr.slice(1);
+  // uppercase characters after spaces
+  return lowerStr.replace(/\W\S/g, t => t.toUpperCase());
+};
```

Comment:
Absolutely brilliant, but I wonder if this is a necessity or an overkill? CSS's `text-transform: capitalize;` could probably work just fine without the need of javascript:

https://developer.mozilla.org/en-US/docs/Web/CSS/text-transform

Reply:
good chance that I overengineered it, but the hovertext is coming back ALLCAPS, does capitalize still work with that? 

ah, I could just make lowercase and then use css to capitalize. 👍🏼